### DISCUSSION TITLE
There is a problem with the base group setup of fp running, thank you for your help

### DISCUSSION THREAD

**Discussion by 12jscvb** [deepmodeling/dpgen#1389] [Q&A]
When dpgen runs to the 02.fp phase, the output file in the task displays the following error: The requested basis set <DZVP-MOLOPT-GTH> for element <AL> was not found in the basis set files <BASIS_MOLOPT> .I've tried a lot of things, but it still doesn't work out. I suspect that the fp section in machine.json is set incorrectly on the Bohrium. The screenshot is as follows:
[outpu.txt](https://github.com/deepmodeling/dpgen/files/13216018/outpu.txt)

![2023-10-31 (3)](https://github.com/deepmodeling/dpgen/assets/145745956/21724d28-52cc-4ccd-a09e-8ee104d2c84b)

**Comment by robinzyb** [comment]
This is about CP2K usage. Please check the official website of cp2k for its usage. 
For the error, it is caused by the missing of basis set `DZVP-MOLOPT-GTH` for element `AL`. Check whether the BASIS SET FILE `BASIS_MOLOPT` contains this basis set `DZVP-MOLOPT-GTH`
